cmd/operatorcli/main.go

```diff
@@ -101,7 +106,8 @@ var commandDeclareAlias = &cli.Command{
 	Description: "Declare the alias in the eochain",
 	Action:      runDeclareAlias,
 	Flags:       []cli.Flag{
-		flag.PassphraseFlag,
+		flag.EcdsaPassphraseFlag,
+		flag.EcdsaAliasedPassphraseFlag,
```

Comment:
should use this flag in the generate-alias command.
changing the EcdsaPrivateKeyFlag in the generate-alias command to be compatible with all other commands is required using this feature

Reply:
fixed in https://github.com/Eoracle/Eoracle-operator-cli/pull/16/commits/ce047e6cb4ebe44a888bc8e066d0a7ec44786656